zbus/src/connection/mod.rs

```diff
@@ -111,6 +122,10 @@ pub(crate) type MsgBroadcaster = Broadcaster<Result<Message>>;
 ///
 /// For sending messages you can either use [`Connection::send`] method.
 ///
+/// To gracefully close a connection while waiting for any outstanding method calls to complete,
+/// see [`Connection::graceful_shutdown`]. To immediately close a connection in a way that will
+/// disrupt any outstanding method calls, see [`Connection::close`].
```

Comment:
* Let's add this sentence: "If you do not need the connection shutdown to be immediate and do not care about outstanding method calls to services, you can simply drop the `Connection` instance.
* nit: "see" -> "use" (in both places).

Reply:
Done, with a little extra about exactly how dropping the Connection behaves.